### PR TITLE
Fast Unstake Support

### PR DESCRIPTION
**Pull request by rossbulat**
- [x] Implemented fast unstake. 
- [x] Leverage local storage to not repeat check eras.
- [x] Simplifed  bond experience by minimising modals.
- [x] Iterated `StatBox` app-wide and refreshed overview announcements.
- [x] more intuitive unstaking

### CODE REVIEW

**Comment by rossbulat** [comment]
Files that need new translation keys @TingALin:

```
src/library/Hooks/useUnstaking/index.tsx
src/modals/Bond/index.tsx
src/modals/ManageFastUnstake/index.tsx
src/modals/Unbond/index.tsx
src/modals/Unstake/index.tsx
src/pages/Nominate/Active/UnstakePrompts.tsx
src/pages/Overview/NetworkSats/Announcements.tsx
src/pages/Overview/NetworkSats/Header.tsx
src/pages/Overview/Stats/SupplyStaked.tsx
src/pages/Pools/Home/ClosurePrompts.tsx
```

**Comment by TingALin** [comment]
const checkedLength = checked.length;
   return `${t('checkingEras', {checkedLength, bondDuration})}...`;
   "checkingEras":"Checking {{checkedLength}} of {{bondDuration}} eras"
   "checkingEras":"正在查验 {{bondDuration}} eras中的 {{checkedLength}}"


Exposed ${lastExposed} Era${lastExposed !== 1 ? `s` : ``} Ago`
return `${t('lastExposed', {count: lastExposed})}`;
"lastExposed_one":"Exposed {{count}} Era Ago"
"lastExposed_other":"Exposed {{count}} Eras Ago"
"lastExposed":"在{{count}}Era前己被显示"

return 'In Queue';
t('inQueue')
"inQueue":"In Queue"
"inQueue":"在队列中"
return 'Fast Unstake';
t('fastUnstake')
"fastUnstake":"Fast Unstake"
"fastUnstake":"快速解除抵押"

(`You have ${totalUnlockChuncks} unlock${ totalUnlockChuncks === 1 ? '' : 's'} active. No unlocks can be active to register for fast unstake. Rebond or withdraw your unlocks to become fully bonded and try registering for fast unstake again.`);
(${t('unlockActive', {count: totalUnlockChuncks})})
"unlockActive_one":"You have {{count}} unlock active. No unlocks can be active to register for fast unstake. Rebond or withdraw your unlocks to become fully bonded and try registering for fast unstake again."
"unlockActive_other":"You have {{count}} unlocks active. No unlocks can be active to register for fast unstake. Rebond or withdraw your unlocks to become fully bonded and try registering for fast unstake again."
"unlockActive":"您有{{count}}己解锁处于活跃状态。无己解锁可被快速解除抵押。重新解除质押或撤回己解锁以完全绑定，然后再次尝试快速解除抵押"

<h2 className="title">{t('wereLastExposed')} {lastExposed}</h2>
"wereLastExposed":"You Were Last Exposed"
"wereLastExposed":"您上次被显示于"

<Trans
defaults={t('toRegisterFastUnstake')  || ''}
components={{ b: <b /> }}
/> 
"toRegisterFastUnstake":"To register for fast unstake, you must <b>not</b> be actively staking for more than {{bondDuration}} eras."
"toRegisterFastUnstake":"如需快速解除抵押, 您必须<b>无</b>活跃抵押长于{{bondDuration}}个eras"

{t('ableToRegisterFastUnstake'{count: erasRemaining})}
"ableToRegisterFastUnstake_one":"If you are inactive for at least {{count}} more era, you will be able to register for fast unstake."
"ableToRegisterFastUnstake_other":"If you are inactive for at least {{count}} more eras, you will be able to register for fast unstake."
"ableToRegisterFastUnstake":"如您在至少{{count}}个era内处于非活动状态,您将快速解除抵押"

{t('registerFastUnstake')}
"registerFastUnstake":"Register Fast Unstake"
"registerFastUnstake":"快速解除抵押"

{t('onceRegisterd')}
"onceRegisterd":" Once registerd you will be waiting in the fast unstake queue."
"onceRegisterd":" 点击快速解除抵押后，您将被列在等待队列中"

{t('numberOfQueue')}
"numberOfQueue":"Number of accounts currently in the fast unstake queue"
"numberOfQueue":"当前处于快速解除抵押队列中的帐户数"

"registeredAndWaiting":"Registered and Waiting to Unstake"
"registeredAndWaiting":"已点击并等待快速解除抵押中"
"exactTimingUnknown":" The fast unstake queue is unordered, so the exact timing of being selected is not known."
"exactTimingUnknown":" 快速解除抵押队列是无序的，因此被选择的确切时间是不确定的"
"cancelFastUnstake":"Cancel Fast Unstake"
"cancelFastUnstake":"取消快速解除抵押"
"Register":"Register"
"Register":"注册快速解除抵押"
"":"Unable to unbond. Your bonded funds are below the minumum of"
"":"无法取消质押。您的债券资金低于"
"":" As the pool depositor you must maintain a bond of at least"
"":" 作为提名池存款人，您必须至少持有"
"":" As a pool member you must maintain a bond of at least"
"":" 作为一名提名池成员，你必须至少持有"
"":" Nominating 10 Validators"
"":" 提名10名验证人"
"":"Minimum Nominate Bond"
"":"最低提名质押"
"":"Unstake"
"":"解除抵押"
"":"Fast Unstake in Progress"
"":"快速解除抵押正在进行中"
"":"Unstake in Progress"
"":"解除抵押正在进行中"
"":"You are in the fast unstake queue. You will not be able to carry out any nominator tasks while you are registered for fast unstake"
"":"您正在快速解除抵押队列中。当注册快速解除抵押时，您将无法执行任何提名人任务"
"":"Waiting for unlocks to become available to withdraw."
"":"等待己解锁变为可提取"
"":"Your bonded funds are now unlocked and ready to withdraw"
"":"您的质押资金现已解锁并可提取"
"":"If you no longer wish to unstake, rebond your {network.unit} and start nominating again"
"":"如果您需要取消解除抵押，请重新质押您的｛network.unit｝并再次开始提名"
"":"Unlocked"
"":"已解锁"
"":"is currently being staked on"
"":"目前正在进行抵押"
"":"The total {{network.unit}} currently being staked amongst all validators and nominators"
"":"当前共有{{network.unit}}在所有验证人和提名人之间做抵押"
"":"Online Validators"
"":"在线验证人"
"":"Active Pools"
"":"活跃提名池"
"":"Supply Staked"
"":"抵押比例"

File: src/locale/cn/library.json [hunk 1]
```diff
@@ -136,6 +136,12 @@
     "viewPoolNominations": "查看池提名",
     "waiting": "等待中",
     "walletNotFound": "未找到钱包",
-    "whenActivelyNominating": "当活跃提名时"
+    "whenActivelyNominating": "当活跃提名时",
+
+    "fastUnstake": "Fast Unstake",
```
Comment (rossbulat) [inline src/locale/cn/library.json:141]: New translations @TingALin

File: src/locale/cn/modals.json [hunk 2]
```diff
@@ -148,6 +146,35 @@
     "withdrawUnlocked": "取出己解锁",
     "youMust_account": "您必须导入帐户才能停止提名",
     "youMust_controller": "您必须导入controller帐户才能停止提名",
-    "chooseLanguage": "选择语言"
+    "chooseLanguage": "选择语言",
+
+    "addToBond": "Add to Bond",
```
Comment (rossbulat) [inline src/locale/cn/modals.json:151]: New translations @TingALin

File: src/locale/cn/pages.json [hunk 3]
```diff
@@ -64,43 +64,51 @@
...
-      "stakingOnNetwork": "共 {{lastTotalStakeBase}} {{networkUnit}}在网络中被活跃抵押中",
       "subscanDisabled": "Subscan己断开",
       "supplyStaked": "抵押比例",
       "totalNominators": "提名人总数",
       "totalNumAccounts": "已加入提名池的帐户总数",
-      "unlocking": "正在解锁"
+      "totalValidators": "所有验证人",
+      "unlocking": "正在解锁",
+
+      "networkCurrentlyStaked": "{{total}} {{unit}} is currently being staked on {{network}}.",
```
Comment (rossbulat) [inline src/locale/cn/pages.json:107]: New translations @TingALin

File: src/locale/cn/pages.json [hunk 4]
```diff
@@ -199,7 +207,9 @@
       "waitingForActiveNominations": "等待有效提名",
       "withdraw": "提取",
       "withdrawFunds": "取款",
-      "withdrawUnlock": "请取出己解锁金额以继续关闭池"
+      "withdrawUnlock": "请取出己解锁金额以继续关闭池",
```
Comment (rossbulat) [inline src/locale/cn/pages.json:210]: New translations @TingALin

**Pull request merged**